# generated by patch-package 6.4.8 on 2022-09-29 13:17:14
#
# command:
#   npx patch-package keycode
#
# declared package:
#   keycode: 2.2.1
#
diff --git a/node_modules/keycode/index.js b/node_modules/keycode/index.js
index 9dc7ce1..7d8025d 100644
--- a/node_modules/keycode/index.js
+++ b/node_modules/keycode/index.js
@@ -137,6 +137,10 @@ var aliases = exports.aliases = {
   'spacebar': 32,
   'pgup': 33,
   'pgdn': 34,
+  'arrowleft': 37,
+  'arrowup': 38,
+  'arrowright': 39,
+  'arrowdown': 40,
   'ins': 45,
   'del': 46,
   'cmd': 91
